app/components/ThemeImage/ThemeImage.tsx

```diff
@@ -64,7 +64,9 @@ export const ThemeImage = ({ image, index, scrollX }: ThemeImageProps) => {
 
         return true;
       }
-    } catch (error) {
+    } catch (error: any) {
+      if (error?.toString().includes('Purchase was cancelled')) return false;
```

Comment:
do it this way because instanceof Error ensures that the error is an instance of the Error class or its subclasses, and avoid using type any.
```suggestion
     if (error instanceof Error && error.message.includes("Purchase was cancelled")) {
        return false;
    }
```

Reply:
That makes sense, thanks :)